### PR TITLE
Fix mask to binary

### LINKED ISSUE #1751: NiftiMasker with target_affine throws exception.
I am using the NifitiMasker in a sklearn pipeline. Everythings works fine until i try to reshape the images:

`nifti_masker = NiftiMasker(target_affine=np.eye(3)*3)`

The error i got was:

`File "/anaconda3/envs/toolbox/lib/python3.6/site-packages/nilearn/masking.py", line 62, in _load_mask_img
    % values)
ValueError: Given mask is not made of 2 values: [-0.0000000e+00  1.4012985e-45  2.8025969e-45 ...  5.1444152e-15
  5.3116801e-15  1.0000000e+00]. Cannot interpret as true or false`

I unsing nilearn version 0.4.2 .

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Fixes #1751

### CODE REVIEW

File: nilearn/input_data/nifti_masker.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # Author: Gael Varoquaux, Alexandre Abraham
 # License: simplified BSD
 
+import numpy as np
```
Comment (GaelVaroquaux) [inline nilearn/input_data/nifti_masker.py]: Why is the numpy import necessary?

Comment (KamalakerDadi) [reply nilearn/input_data/nifti_masker.py]: May be I have forgotten to remove while removing sanity checks. I will clean up in next commit along with some smoke tests.

Thanks

**Comment by KamalakerDadi** [comment]
Seems like travis is not happy. I will look into that

**Comment by GaelVaroquaux** [comment]
I am not convinced that this is the proper fix.

Have you managed to reproduce the bug on a small example?

**Comment by KamalakerDadi** [comment]
Ok. I am closing this one.

Yes nearest interpolation could be an option.

**Pull request closed without merging**